peps/pep-0777.rst

```diff
@@ -0,0 +1,304 @@
+PEP: 777
+Title: How to Re-invent the Wheel
+Author: Ethan Smith <ethan@ethanhs.me>
+Sponsor: Barry Warsaw <barry@python.org>
+PEP-Delegate: Paul Moore <p.f.moore@gmail.com>
+Status: Draft
+Type: Standards Track
+Topic: Packaging
+Created: 09-Oct-2024
+Post-History:
+
+Abstract
+========
+
+The current :pep:`wheel 1.0 specification <427>` was written over a decade ago,
+and has been extremely robust to changes in the Python packaging ecosystem.
+Previous efforts to improve the wheel specification 
+:pep:`were deferred <491#pep-deferral>` to focus on other packaging
+specifications. Meanwhile, the use of wheels has changed dramatically in the
+last decade. There have been many requests for new wheel features over the
+years; however, a fundamental obstacle to evolving the wheel specification has
+been that there is no defined process for how to handle adding
+backwards-incompatible features to wheels. Therefore, to enable other PEPs to
+describe new enhancements to the wheel specification, this PEP prescribes
+compatibility requirements on future wheel revisions. This PEP does *not*
```

Comment:
Consider bolding this portion of the sentence to give the reader an "at-a-glance" perspective of the PEP.

```suggestion
describe new enhancements to the wheel specification, **this PEP prescribes**
**compatibility requirements on future wheel revisions**. This PEP does *not*
```